index.bs

```diff
@@ -4902,6 +4991,55 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the authenticator's credential created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean credProps;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the dictionary of discovered credential properties as reported by either the authenticator itself or by the 
+    client if known, such as when a platform authenticator is used. This dictionary is initially defined with one credential
+    property rk to indicate if the authenticator has created or used a resident key credential. Other credential properties
```

Comment:
`rk` -> `{{AuthenticationExtensionsClientInputs/rk}}`

Reply:
Fixed in subsequent commit by it's actually: `rk -> {{CredentialPropertiesOutput/rk}}`

Reply:
Oh yeah, good catch!